mozilla/fx-private-relay
# MPP-2822: add gtag to Layout and update gaEvent hook to ping both GA endpoints

# Add GA4
## Screenshot

![image](https://github.com/mozilla/fx-private-relay/assets/71928/ebc3f45f-979a-4b3e-85f2-0804842a6401)


## How to test

1. Checkout this branch
2. In `frontend/` run `npm run dev`
3. Go to http://localhost:3000/
   * [ ] In the network inspector, verify a request to www.googletagmanager.com is sent
5. Go to https://analytics.google.com/analytics/web/#/p314403930/realtime/overview
   * [ ] Verify you see your traffic in the realtime report
6. In `frontend/` run `npm run build`
7. In app root, run `python manage.py collectstatic`
8. In app root, run `python manage.py runserver`
9. Go to http://127.0.0.1:8000/
   * [ ] In the network inspector, verify a request to www.googletagmanager.com is sent
10. Go to https://analytics.google.com/analytics/web/#/p314403930/realtime/overview
    * [ ] Verify you see your traffic in the realtime report

# Checklist (Definition of Done)
